specs/svg-native/index.bs

```diff
@@ -64,62 +70,118 @@ The file extension for SVG Native is <code>.svn</code>. The UTI for SVG Native i
+The ''scroll'' and the ''auto'' values of the 'overflow' property are not supported by SVG Native.
 
 Document Structure {#struct}
 ============================
 
-The root element must be an <{svg}> element, and all other <{svg}> elements are forbidden.
+The root element must be an <{svg}> element, and all other <{svg}> elements are not supported by SVG Native.
+<!-- https://www.w3.org/2019/07/15-svg-minutes.html -->
+
+Conditional processesing attributes and elements are not supported in SVG Native. These include the <{switch}> element.
+
+<!-- There are too many things which are unobservable in SVG Native. Don't need to list them all out. -->
 
-Section 5.8 "Conditional Processing" is deleted.
+The <{meta}> and <{metadata}> elements are not supported in SVG Native.
+
+Elements in foreign namespaces, except the xlink (<code>http://www.w3.org/1999/xlink</code>) namespace, are not supported in SVG Native.
 
 <{image}> elements must only contain base64-encoded <code>data:</code> URLs of [[!JPEG]] or [[!PNG]] images. [[!APNG]] images must be rendered without animation, using standard backward-compatibility with static [[!PNG]] images. All other image formats must be ignored.
 
 All external resource loading is forbidden.
 
+The <{symbol}> element is not supported by SVG Native.
+<!-- https://www.w3.org/2019/07/15-svg-minutes.html -->
+
 Styling {#styling}
 ==================
 
-The <{style}> element and the 'style' attribute are forbidden.
+The <{style}> element and the 'style' attribute is not supported in SVG Native.
+<!-- https://www.w3.org/2019/05/06-svg-minutes.html -->
 
-Using ''calc()'', ''env()'', or ''var()'' is forbidden.
+''calc()'' is not supported by SVG Native.
 
-Note: Other environments may relax these rules and allow these functions as necessary. For example, the [[!OPENTYPE]] spec might allow ''env()'' or ''var()'' in color fonts to support font color palettes.
+''env()'' and ''var()'' are supported by SVG Native.
+<!-- https://www.w3.org/2019/05/06-svg-minutes.html -->
+<!-- https://www.w3.org/2019/05/20-svg-minutes.html -->
 
-The CSS 'all' property is forbidden.
+The CSS 'all' property is not supported by SVG Native.
 
-All the global CSS keywords are forbidden. These include:
+All the global CSS keywords are not supported by SVG Native. These include:
 
 1. ''initial''
 2. ''inherit''
 3. ''unset''
 4. ''revert''
 
+Percentage length values are not supported by SVG Native.
+<!-- https://www.w3.org/2019/07/01-svg-minutes.html -->
+
+<a lt="relative length">Relative units</a> are not supported by SVG Native.
+<!-- https://www.w3.org/2019/07/01-svg-minutes.html -->
+
+Geometry Properties {#geometryproperties}
+=========================================
+
+The grammar for the 'cx', 'cy', 'r', 'rx', 'ry', 'x', 'y', 'width', and 'height' properties is:
+
+<code><<number>> | <<length>> | <<percentage>></code>
```

Comment:
What is the purpose of re-writing the grammar here?

Percentages are already forbidden above.

The allowance of "number" comes out of the SVG rules on parsing presentation attributes, although it might be useful to add a clarifying note about it. (But note that the parsing rule also affects other presentation attributes, e.g., stroke-width.) It's interpretation as a px value is likewise standard SVG.

I think it would be more consistent with the rest of the spec to forbid "keyword values" for these properties rather than writing a new grammar.